cmake/MacroCreateBundle.cmake

```diff
@@ -128,31 +132,32 @@ if(${PROJECT_NAME}_RESOURCES)
   if(${PROJECT_NAME}_RESOURCES)
     set(_wd ${CMAKE_CURRENT_SOURCE_DIR}/resources)
   endif()
-  usFunctionAddResources(TARGET ${${PROJECT_NAME}_TARGET}
-                         WORKING_DIRECTORY ${_wd}
-                         FILES ${${PROJECT_NAME}_RESOURCES}
-                        )
+  FunctionAddResources(TARGET ${${PROJECT_NAME}_TARGET}
+                       WORKING_DIRECTORY ${_wd}
+                       FILES ${${PROJECT_NAME}_RESOURCES}
+                      )
 endif()
 if(${PROJECT_NAME}_BINARY_RESOURCES)
-  usFunctionAddResources(TARGET ${${PROJECT_NAME}_TARGET}
-                         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/resources
-                         FILES ${${PROJECT_NAME}_BINARY_RESOURCES}
-                        )
+  FunctionAddResources(TARGET ${${PROJECT_NAME}_TARGET}
+                       WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/resources
+                       FILES ${${PROJECT_NAME}_BINARY_RESOURCES}
+                      )
 endif()
-usFunctionEmbedResources(TARGET ${${PROJECT_NAME}_TARGET})
+FunctionEmbedResources(TARGET ${${PROJECT_NAME}_TARGET})
 
 #-----------------------------------------------------------------------------
 # Install support
 #-----------------------------------------------------------------------------
 
 if(NOT US_NO_INSTALL)
   install(TARGETS ${${PROJECT_NAME}_TARGET}
-          EXPORT us${PROJECT_NAME}Targets
+          EXPORT ${PROJECT_NAME}Targets
           RUNTIME DESTINATION ${RUNTIME_INSTALL_DIR} ${US_SDK_INSTALL_COMPONENT}
           LIBRARY DESTINATION ${LIBRARY_INSTALL_DIR} ${US_SDK_INSTALL_COMPONENT}
-          ARCHIVE DESTINATION ${ARCHIVE_INSTALL_DIR} ${US_SDK_INSTALL_COMPONENT}
-          PUBLIC_HEADER DESTINATION ${HEADER_INSTALL_DIR} ${US_SDK_INSTALL_COMPONENT}
-          PRIVATE_HEADER DESTINATION ${HEADER_INSTALL_DIR} ${US_SDK_INSTALL_COMPONENT})
+          ARCHIVE DESTINATION ${ARCHIVE_INSTALL_DIR} ${US_SDK_INSTALL_COMPONENT})
+
+  install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/cppmicroservices DESTINATION ${HEADER_INSTALL_DIR})
+  install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/include/cppmicroservices DESTINATION ${HEADER_INSTALL_DIR})
```

Comment:
Same question as above, was there a problem with the `PUBLIC_HEADER` target property?